src/emcpy/plots/create_plots.py

```diff
@@ -1,13 +1,16 @@
 # This work developed by NOAA/NWS/EMC under the Apache 2.0 license.
 import os
+import emcpy
```

Comment:
why are we importing emcpy wholesale here? Should do like what is done at the end of the `import` list

Reply:
Good question. I was taking a look at how the old version got the path needed to get to the logo `.png` files, and it had `emcpy.emcpy_directory` (line 333), so used that. Is there a better approach to point to the directory with the logo figures?

Reply:
then maybe just do something like `import emcpy.emcpy_directory as emcpydir` or something like that?